Mathlib/RingTheory/Kaehler.lean

```diff
@@ -388,10 +388,10 @@ theorem KaehlerDifferential.End_equiv_aux (f : S →ₐ[R] S ⊗ S ⧸ KaehlerDi
     exact e₁.symm.trans (e.trans e₂)
 #align kaehler_differential.End_equiv_aux KaehlerDifferential.End_equiv_aux
 
-set_option maxHeartbeats 4400000 in
+set_option maxHeartbeats 7000000 in
```

Comment:
Are those still needed with the recent speed up?

Reply:
Yes, the old limit doesn't work. I'll use `count_heartbeats` again to squeeze it a bit, but it will take 10 minutes or so for me to have a result.